README.md

```diff
@@ -38,21 +38,29 @@ To customize your test suite configuration you can add a few more options to php
     <server name="KERNEL_DIR" value="/path/to/dir/with/kernel" />
     <server name="EXPECTED_RESPONSE_DIR" value="/path/to/expected/responses/" />
     <server name="MOCKED_RESPONSE_DIR" value="/path/to/mocked/responses/" />
+    <server name="FIXTURES_DIR" value="/../src/DataFixtures/ORM/" />
```

Comment:
I would suggest to be consistent with the previous examples, so `value="/path/to/DataFixtures/"`